CMS-Enterprise/mint-app
# [NOREF] Sidepanel Component Consolidation

# MINT-0000

## Description

There are repeated uses of the React Modal to be reused as a panel that slides in (e.g. Discussion and Operation Solution help articles). This PR is to consolidate that side panel, so it is easily referable and modifiable. 


## How to test this change

1.  Open Discussion panels and verify that it is still working as expected
2. Open any operational solution and see that side panel is still working as expected
3. Check responsiveness

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
